### DISCUSSION TITLE
Supabase issue TypeError: Cannot destructure property 'PostgrestClient' of '_cjs_index_js__WEBPACK_IMPORTED_MODULE_0___default(...)' as it is undefined.

### DISCUSSION THREAD

**Discussion by chanceDira** [supabase/supabase#27675] [Questions] [answered]
I got this error when I re-installed the packages, before everything was working fine.

here my version     "@supabase/supabase-js": "^2.43.4",

**Comment by yathindrak** [comment]
Getting the same issue with plasmo react setup.

**Comment by soedirgo** [comment, marked as answer]
Hi! This should be fixed as of supabase-js v2.44.3.
